docs/journalist.rst

```diff
@@ -375,16 +518,17 @@ such as an editor's) and click **OK**.
 |Encrypting 2|
 
 When you are done encrypting, you will have another document with the
-same filename but ending in ".gpg". This file is encrypted to the GPG
+same filename but ending in ``.pgp`` (not ``.gpg``; the ``.pgp`` extension is
+just another way to refer to the same format). This file is encrypted to the GPG
 keys you selected. You can safely copy these encrypted files to the
-*Transfer Device* to transfer them to your workstation.
+*Transfer Device* to transfer them to your everyday workstation.
```

Comment:
Definitely worth reiterating the risks of opening random attachments here

Reply:
Added an "Important" warning re: malware, reiterating earlier points.